### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by ywq2001** [CodeEditorBench/CodeEditorBench#9]
The data set you provided seems to have no problem description. Do you have problem description for each problem

**Comment by Jiawei-Guo** [comment]
For the problem description of the dataset, you can see subsection 3.1 of our arxiv paper: https://arxiv.org/pdf/2404.03543.

**Comment by ywq2001** [comment]
Sorry, maybe I didn't make it clear. I meant to say that for each piece of data in this code debug data set, you have provided incorrect_solution and public_test_cases. Have you also collected problem descriptions and could you please provide them to me

**Comment by ywq2001** [comment]
In addition, for the task of code debug, I used the solutions in the data set and the evaluation system you provided for testing, and found that the correct rate could not reach 100%. Do you know this situation

**Comment by ywq2001** [comment]
![336044241-6b8deefa-0bab-42e9-bba0-835def084988](https://github.com/CodeEditorBench/CodeEditorBench/assets/143774034/f68e4f82-16c5-4b57-9b35-0f45cbdfd500)
I used 100 pieces of data in the code debug data set provided by you to test, and the result is shown in the figure

**Comment by Jiawei-Guo** [comment]
The output of all test cases is generated based on the correct code. Is it because you are not doing add_template on the correct code? Due to the special nature of the OJ system, all code needs to go through the add template step, and I'm guessing that's what's causing it

**Comment by ywq2001** [comment]
But all of my evaluations followed the steps given, and I added templates
![图片](https://github.com/CodeEditorBench/CodeEditorBench/assets/143774034/7b96094a-aaac-41e8-933c-448e395e4dd3)

**Comment by ywq2001** [comment]
运行add_template.py得到上面图中的内容

**Comment by ywq2001** [comment]
The above result is obtained by running add template.py

**Comment by Jiawei-Guo** [comment]
Thank you for pointing out the problem, we will verify it locally and get the result to you as soon as possible.

**Comment by ywq2001** [comment]
Thank you, I hope you can verify it as soon as possible.

**Comment by Jiawei-Guo** [comment]
![image](https://github.com/CodeEditorBench/CodeEditorBench/assets/74909365/8e5aa144-b0f5-4bda-b10d-76c518ef51d7)
We locally evaluated the source code and it looks like it meets expectations, next week we'll compare the code stored in the database with the released code, there may be a problem with the problem id mapping.

**Comment by ywq2001** [comment]
There may be a problem with the docker evaluation system you provided. I used deepseek-coder-6.7b-inst to conduct codedebug tests, and I found that all questions with Python3 language were judged to be wrong, and the cause of the error was wrong answer. But the final pass@ is 28.8, which is outrageous.
![图片](https://github.com/CodeEditorBench/CodeEditorBench/assets/143774034/0ae136b2-c043-45a8-9950-4da476fbbc7e)
